pom.xml

```diff
@@ -22,13 +22,22 @@
     </repository>
   </repositories>
 
-  <artifactId>fcrepo-swordserver</artifactId>
+  <artifactId>fcrepo4-swordserver</artifactId>
   <name>Fedora Repository SWORD Provider</name>
   <description>
     Service implementing a SWORD endpoint for Fedora Commons Repositories.
   </description>
   <packaging>bundle</packaging>
 
+  <scm>
+    <connection>scm:git:git://github.com/fcrepo4-labs/${project.artifactId}.git
+    </connection>
+    <developerConnection>scm:git:git@github.com:fcrepo4-labs/${project.artifactId}.git
+    </developerConnection>
+    <url>https://github.com/fcrepo4-labs/fcrepo4-swordserver</url>
```

Comment:
These are the projects that are currently involved in the release process. If you would like to create at ticket, it seems constructive.
* fcrepo-build-tools
* fcrepo4
* fcrepo-message-consumer 
* fcrepo-module-auth-rbacl 
* fcrepo-module-auth-xacml 
* fcrepo-camel 
* ontology 
* fcrepo4-client 
* fcrepo-audit 
* fcrepo-camel-toolbox 
* fcrepo-webapp-plus 
* fcrepo4-release-tests 
* fcrepo4-oaiprovider 
fcrepo4-swordserver 
migration-utils